int64 time
bool value
